src/main/java/org/apache/maven/plugins/jlink/AbstractJLinkToolchainExecutor.java

```diff
@@ -0,0 +1,204 @@
+package org.apache.maven.plugins.jlink;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.commons.lang3.SystemUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.logging.Log;
```

Comment:
use SLF4J

Reply:
I was not going to refactor existing code. Is prefer to do this in another PR.

Reply:
Orig: https://github.com/apache/maven-jlink-plugin/blob/5e2b1272727400d81eb64a3de50557f343097319/src/main/java/org/apache/maven/plugins/jlink/JLinkMojo.java#L283
etc